hw03_barkovskan.py

```diff
@@ -0,0 +1,41 @@
+import turtle
+
+def drawsquare(bubbles, size):
+    """Draw a square using turtle."""
+    for _ in range(4):
+        bubbles.forward(size)
+        bubbles.left(90)
+
+def draw_row(bubbles, color1, color2):
+    """Draw a row of squares with alternating colors."""
+    for _ in range(8):
+        if _ % 2 == 0:
+            bubbles.fillcolor(color1)
+        else:
+            bubbles.fillcolor(color2)
+        bubbles.begin_fill()
+        drawsquare(bubbles, 60)
+        bubbles.forward(60)
+        bubbles.end_fill()
+
+def main():
+    """Main function to set up turtle and draw the pattern."""
+    wn = turtle.Screen()
+    wn.bgcolor("black")
+    bubbles = turtle.Turtle()
+    bubbles.speed(0)
+
+    positions = [100, 40, -20, -80, -140, -200]
+    colors1 = ["pink", (0.165, 0.42, 0.42), "orange", (0.80, 0.203, 0.118), "yellow", (0.143, 0.188, 0.143)]
+    colors2 = [(0.245, 0.245, 0.220), "white", (0.237, 0.145, 0.33), "blue", (0.85, 0.107, 0.47), "purple"]
+
+    for pos, color1, color2 in zip(positions, colors1, colors2):
```

Comment:
This was definitely looked up from an external resource. You MUST cite your sources moving forward, or we will have academic dishonesty problems!